homeassistant/components/climate/honeywell.py

```diff
@@ -66,7 +79,7 @@ def _setup_round(username, password, config, add_devices):
                 [RoundThermostat(evo_api, zone['id'], i == 0, away_temp)]
             )
     except socket.error:
-        _LOGGER.error(
+        _LOGGER.exception(
```

Comment:
We should not use `.exception`. This will cause a stacktrace to be printed for the users although the error message already tells them what is going on.